src/main/jbake/content/blog/2017/05/31/hawkular-grafana-ootb.adoc

```diff
@@ -0,0 +1,253 @@
+= Hawkular and Grafana Out of the Box
+Joel Takvorian
+2017-05-31
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, metrics, grafana, docker, vertx
+:figure-caption!:
+
+link:https://github.com/jotak/hawkular-grafana-outofthebox[Hawkular and Grafana Out of the Box] is a new way to quickly setup Hawkular and Grafana together with very little work.
+
+Assuming you already have docker installed and recent enough (you need at least Compose 1.6.0 and Docker Engine 1.10.0),
+you're just 2 lines of YAML edition and a `docker-compose up` away from having Hawkular + Grafana running and pre-configured.
+
+It is NOT intended for production/long-term usage unless you do some extra work to make it secured.
```

Comment:
I would not add this line; this is not related to the technical details presented here.